lib/balance-tx/lib/Cardano/Tx/Balance.hs

```diff
@@ -0,0 +1,7 @@
+-- |
+-- Copyright: © 2022 IOHK
+-- License: Apache-2.0
+--
+module Cardano.Tx.Balance
```

Comment:
I think I'd lean towards `Cardano.Write.Tx.Balance` and us renaming `Cardano.Wallet.Write.Tx` to `Cardano.Write.Tx`

Reply:
But can be reconciled/discussed later

Reply:
> I think I'd lean towards Cardano.Write.Tx.Balance and us renaming Cardano.Wallet.Write.Tx to Cardano.Write.Tx
> But can be reconciled/discussed later

Definitely! We can revise this as we go later on. I'm happy to go with your suggestion, though I wonder if it still makes sense outside of the context of the wallet, where we do have a clear dichotomy between `Read` and `Write`?

In the `balance-tx` library, we (presumably) only ever deal with transactions that are intended to be "written" to the blockchain, so perhaps it's superfluous to specify it in the module hierarchy?

But on the other hand, if `balance-tx` is intended to exist within a family of libraries that collectively deal with reading and writing transactions, then nesting the main module within `Write` might make sense. 👍🏻

Reply:
Two arguments for including `Write`:
- It is a key concept that will be permeating the entire library.
- It is a pretty unique identifier (as opposed to `Cardano.Tx` ~ `Cardano.Api.Tx`)
    - Can use `WriteTx` as qualified name as opposed to... overloaded `Tx`?

We currently have
```haskell
import qualified Cardano.Wallet.Write.Tx as WriteTx
```
and I'm imagining:

```haskell
import qualified Cardano.Write.Tx as WriteTx -- either this
import qualified Cardano.Api.Write.Tx as WriteTx -- or this for cardano-api-ified version
-- ^ Motivation Cardano.Write.Tx.CardanoApi is ugly

import Cardano.Write.Tx.Balance (balanceTx)
```